### ISSUE TITLE
Add: bigi

### ISSUE THREAD

**Issue by ga250182** [cdnjs/packages#1136]
### Library name

bigi

### Library license

https://github.com/cryptocoinjs/bigi/blob/master/LICENSE.md

### Library source

http://cryptocoinjs.com/modules/misc/bigi/

### Files to include

All files

**Comment by MattIPv4** [comment]
👋 This library makes use of Node.js `require` calls that won't work in the browser. For us to ship a library on the CDN that will work on the web, the library needs to be distributed with a version that will work on the web.

**Comment by ga250182** [comment]
Thanks Matt, Wanted to check if crypto lib is part of cdnjs. I was not able to see crypto node JS lib

**Comment by MattIPv4** [comment]
No, we don't distribute Node.js libraries, we distribute libraries built for the web.

**Comment by MattIPv4** [comment]
Yup, looks like we have https://cdnjs.com/libraries/crypto-js
